docs/docs/index.md

```diff
@@ -134,6 +134,39 @@ with q.connection.pipeline() as pipe:
 
 `Queue.prepare_data` accepts all arguments that `Queue.parse_args` does.
 
+### Grouping jobs
+_New in version 1.16.0._  
+Multiple jobs can be added to a Group to allow them to be tracked by a single ID:
+
+```python
+from rq import Queue
+from rq.group import Group
+
+group = Group.create(connection=redis_conn)
```

Comment:
I wonder if we should allow (or force) the user to pass in a `name` argument.

Reply:
create (and group constructor) does accept an optional ID, I did not make it required to stay consistent with the Job API.